### PR TITLE
Special Blockquotes

### PR DESCRIPTION
**Pull request by ulivz**
# Summary 

Finished https://github.com/egoist/docute/issues/151

And an important thing is that with markdown-it, we doesn't need `<!-- -->` between two blockquotes.

### CODE REVIEW

**Comment by egoist** [comment]
Deploy preview for *docute* ready!

Built with commit 37a1e9be3ee3c0faf18f1a4ddc8df4516dc02165

https://deploy-preview-157--docute.netlify.com

File: packages/docute/src/utils/markdown-it/messageBlocks.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        const blockquoteOpenToken = tokens[idx]
+        const blockquoteCloseToken = tokens[idx + 4]
+        blockquoteOpenToken.tag = 'div'
+        blockquoteOpenToken.attrs = [['class', `message ${type.toLowerCase()}`]]
+        blockquoteCloseToken.tag = 'div'
+      }
+    }
+    return self.renderToken(...args)
+  }
+}
```
Comment (egoist) [inline packages/docute/src/utils/markdown-it/messageBlocks.js]: maybe add some tests at `./messageBlocks.test.js`

File: website/docs/essentials/special-blockquotes.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (egoist) [inline website/docs/essentials/special-blockquotes.md]: Put all the markdown stuffs at `essentials/markdown-features.md`

Comment (ulivz) [reply website/docs/essentials/special-blockquotes.md]: fixed at 98efc40

File: packages/docute/src/utils/markdown-it/messageBlocks.js (hunk 1 above)
Comment (ulivz) [reply packages/docute/src/utils/markdown-it/messageBlocks.js]: fixed at 9a58389, thanks

**Comment by ulivz** [comment]
@egoist thank for your clean up and simplification. 👍  
next time I should do more self code review.

**Pull request merged**